src/XrdSciTokens/XrdSciTokensAccess.cc

```diff
@@ -31,6 +32,13 @@
 XrdVERSIONINFO(XrdAccAuthorizeObject, XrdAccSciTokens);
 XrdVERSIONINFO(XrdAccAuthorizeObjAdd, XrdAccSciTokens);
 
+namespace XrdGlobal {
+// The XrdOucEnv object passed to the plugin has a pointer called `XrdTLSContext*`;
+// however, that's a cloned pointer without any of the verification data (CA file or
+// directory), which is precisely what we need.  So, we use the XrdGlobal version.
```

Comment:
Sounds like we should rather solve this issue by passing `XrdOucEnv` with a `XrdTLSContext*` that has the verification data, no? This works around the intended interfaces and creates a hidden connection between `XrdSciTokens` and `XrdTlsContext` which we should try to avoid. Let's hear what @abh3 thinks.

Reply:
Switched over to the new interface... resolving conversation.